handlers/easy_info/__init__.py

```diff
@@ -0,0 +1,12 @@
+from eventhandler import EventHandler
+
+
+class EasyInfoHandler(EventHandler):
+    def on_issue_labeled(self, api, payload):
```

Comment:
I looked into the webhook logs of one repo I maintain and it looks like GitHub actually sends separate `opened` and `labeled` (one per label) payloads when an issue is opened.